warehouse/query-core/src/main/java/datawave/query/iterator/QueryIterator.java

```diff
@@ -1094,9 +1097,12 @@ protected Iterator<Entry<Key,Document>> mapDocument(SortedKeyValueIterator<Key,V
             log.trace("mapDocument " + fieldIndexSatisfiesQuery);
         }
         if (fieldIndexSatisfiesQuery) {
+            //  @formatter:off
             final KeyToDocumentData docMapper = new KeyToDocumentData(deepSourceCopy, this.myEnvironment, this.documentOptions, getEquality(),
-                            getEvaluationFilter(), this.includeHierarchyFields, this.includeHierarchyFields).withRangeProvider(getRangeProvider())
-                                            .withAggregationThreshold(getDocAggregationThresholdMs());
+                            getEventFilter(), this.includeHierarchyFields, this.includeHierarchyFields)
```

Comment:
What happened to using the evaluation filter?

Reply:
The evaluation filter was never used in the QueryIterator, it was only used by the TLDQueryIterator. This change allows the standard QueryIterator to do field filtering (and field based seeking).

You can see later in the change set [where the TLDQueryIterator](https://github.com/NationalSecurityAgency/datawave/pull/2538/files#diff-dcb340cebaade3ad6aa598bd3737a93f7050cde082910d3294d1d8220a9be591R141-R150) intercepts `getEventFilter()` and swaps in the evaluation filter:

Reply:
ah, I see now.